autoscheduler/frontend/src/redux/actions/schedules.ts

```diff
@@ -101,6 +101,8 @@ ThunkAction<Promise<void>, RootState, undefined, ReplaceSchedulesAction | Select
           honors: isBasic ? (courseCard.honors ?? filterDefault) : filterDefault,
           remote: isBasic ? (courseCard.remote ?? filterDefault) : filterDefault,
           asynchronous: isBasic ? (courseCard.asynchronous ?? filterDefault) : filterDefault,
+          // TODO something about full sections goes here?
+          includeFull: isBasic ? (courseCard.includeFull) : false,
```

Comment:
You should remove this comment

I also noticed that this line makes it so that section select won't include full sections unless you select sections manually. I don't know exactly how I feel about this since while I assume most users would prefer this, it makes the behavior different when no sections vs. all sections are selected. We can keep it as-is for now but I just wanted to point this out and maybe discuss it in the next meeting